.ci.yaml

```
@@ -2478,6 +2501,19 @@ targets:
       - bin/**
       - .ci.yaml
 
+  - name: Mac_arm64_ios build_ios_framework_module_test
+    recipe: devicelab/devicelab_drone
+    presubmit: false
+    bringup: true # https://github.com/flutter/flutter/issues/87508
+    timeout: 60
+    properties:
+      tags: >
+        ["devicelab","ios","mac","arm64"]
+      task_name: build_ios_framework_module_test
+    runIf:
+      - dev/**
```

`runIf` should match their `x86` equivalent 
```suggestion
      - dev/**
      - packages/flutter_tools/**
      - bin/**
      - .ci.yaml
```

---

Updated all builders to be consistent with x86.